api/db-migrations/000011_add_pyfunc_config_env.up.sql

```diff
@@ -0,0 +1,10 @@
+-- Migrate pyfunc_config data from old schema to new schema (with new 'env' field set as an empty array by default)
+UPDATE ensembler_configs
+SET pyfunc_config = (SELECT json_build_object(
+        'env', '[]'::jsonb,
```

Comment:
Not sure if this is really necessary since the only pyfunc ensemblers that have been created with this field are the ones in dev, but for consistency's sake I thought having a migration script for this would make things neater. Alternatively, we could consider implementing some lazy migration of the saved pyfunc ensemblers in the API?

Reply:
Actually I feel it's better to not have this in the migration, just to keep the migration history lean.

This feature is still under development. :) We don't really need to worry since we haven't cut any release with migration file # 10 (and as a consequence, we also haven't released this feature to our internal users, as you've noted). We can run this data migration manually on the internal dev DB.